### PR TITLE
Added 24 mainnet and 5 matic tokens

### PR DESCRIPTION
**Pull request by tytarman**
I added some new tokens to our list. I mostly just copied the same formatting from previous adds, and I drew all of my info from CoinGecko. Wasn't sure what token names were best to use so I defaulted to whatever CoinGecko or their websites used. Also, I pulled the token images from the SushiSwap repo when I could as opposed to the tiny CoinGecko images.

I only added the Matic tokens when they were verified on CoinGecko and available to exchange through something like Zapper.

DG has an updated contract address, so I added a new item on the list for the new address and converted the previous address to DG (Old).

### CODE REVIEW

**Comment by tytarman** [comment]
@0xModene just let me know if you want me to tweak something and I'll get right on it

File: package.json [hunk 1]
```diff
@@ -79,5 +79,8 @@
   "publishConfig": {
     "access": "public",
     "registry": "https://registry.npmjs.org/"
+  },
```
Comment (0xModene) [inline package.json:82]: Can you remove this? No need for a dependency here

**Review by 0xModene — CHANGES_REQUESTED** [review]
small nit, but otherwise looks good

File: package.json (hunk 1 above)
Comment (tytarman) [reply package.json]: lol not even sure how that got changed since i didn't even open that file....but cool. Should be fixed now

**Review by 0xModene — APPROVED** [review]
Lgtm

**Pull request merged**